Makefile

```diff
@@ -32,13 +32,10 @@ govendor:
 
 release: upload-release check-release
 
-SELFUPDATE_TMPDIR=.tmp-selfupdate
```

Comment:
Would you confirm that is ok remove this line as well as 40 & 41 ?
/cc: @beyang